src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/actionpostponementrequest/ActionPostponementRequestAboutToSubmitHandler.java

```diff
@@ -108,7 +108,7 @@ private void grantPostponement(SscsCaseData sscsCaseData, PostponementRequest po
 
     private void setHearingDateToExcludedDate(SscsCaseData sscsCaseData, PreSubmitCallbackResponse<SscsCaseData> response) {
         final Optional<Hearing> optionalHearing = emptyIfNull(sscsCaseData.getHearings()).stream()
-                .filter(h -> h.getValue().getHearingDateTime().isAfter(LocalDateTime.now()))
+                .filter(hearing -> LocalDateTime.now().isBefore(hearing.getValue().getStart()))
```

Comment:
Wont this give us all the hearings that have already expired / on previous days ?

Reply:
From what I understand this was done on purpose, but I'll take a closer look as to the rationale behind it and get back to you.

Reply:
Again, I think we should be using getLatestHearing so that we have consistency on this with all our S&L work.